extensions/amp-story-interactive/0.1/amp-story-interactive-abstract.js

```diff
@@ -250,6 +250,8 @@ export class AmpStoryInteractive extends AMP.BaseElement {
     devAssert(this.element.children.length == 0, 'Too many children');
 
     // Initialize all the services before proceeding, and update store with state
+    this.clientIdService_ = Services.cidForDoc(this.element);
```

Comment:
This is not the `clientIdService_` but a promise that will return it

Reply:
Yes but `cidForDoc` calls `getAmpDoc` synchronously, being a promise doesn't change anything in this case.

Reply:
The `clientIdService_` name is confusing since it doesn't contain the service, but a promise. You should either make it be the service, or rename the property.

Reply:
Yeah I agree, I thought you were asking about this not calling `getAmpDoc`.

Since it was being used only in one place, there's no need to store it as a private variable. The `this.clientIdPromise_` is the promise that we want, so I removed the private variable and just access the service promise when we want to get the `clientId`.